doc/classes/EditorSettings.xml

```diff
@@ -1118,13 +1226,25 @@
 		<member name="text_editor/help/show_help_index" type="bool" setter="" getter="">
 			If [code]true[/code], displays a table of contents at the left of the editor help (at the location where the members overview would appear when editing a script).
 		</member>
+		<member name="text_editor/help/sort_functions_alphabetically" type="bool" setter="" getter="">
+		</member>
+		<member name="text_editor/script_list/group_help_pages" type="bool" setter="" getter="">
+		</member>
+		<member name="text_editor/script_list/list_script_names_as" type="int" setter="" getter="">
+		</member>
+		<member name="text_editor/script_list/script_temperature_enabled" type="bool" setter="" getter="">
+		</member>
+		<member name="text_editor/script_list/script_temperature_history_size" type="int" setter="" getter="">
+		</member>
 		<member name="text_editor/script_list/show_members_overview" type="bool" setter="" getter="">
 			If [code]true[/code], displays an overview of the current script's member variables and functions at the left of the script editor. See also [member text_editor/script_list/sort_members_outline_alphabetically].
 		</member>
 		<member name="text_editor/script_list/sort_members_outline_alphabetically" type="bool" setter="" getter="">
 			If [code]true[/code], sorts the members outline (located at the left of the script editor) using alphabetical order. If [code]false[/code], sorts the members outline depending on the order in which members are found in the script.
 			[b]Note:[/b] Only effective if [member text_editor/script_list/show_members_overview] is [code]true[/code].
 		</member>
+		<member name="text_editor/script_list/sort_scripts_by" type="int" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="text_editor/script_list/sort_scripts_by" type="int" setter="" getter="">
			Specifies sorting used for Script Editor's open script list.
		</member>
```

Reply:
The methods are: `Last Edited,Name,Path` should we include that somehow?

Reply:
The sorting can be selected as a dropdown, so it doesn't need to be mentioned in the description. Unless you want to clarify the options.

Reply:
What about:
```suggestion
		<member name="text_editor/script_list/sort_scripts_by" type="int" setter="" getter="">
			Specifies sorting used for Script Editor's open script list. Either by most recently edited, name of the script, or the full path.
		</member>
```
Unsure about the meaning of "path" vs "name"

Reply:
That's good, I'll leave it simple